app/processing/invoice-line/validate-invoice-line.js

```diff
@@ -0,0 +1,15 @@
+const schema = require('./schema')
+
+const validateInvoiceLine = (grossValueInvoiceLine, fundingCode, paymentRequestId) => {

```

Comment:
Could rename the first parameter `invoiceLine` as the schema applies to the full invoice line we're returning which includes elements of gross and reductions